pyrelational/data/data_manager.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         :param hit_ratio_at: optional argument setting the top percentage threshold to compute hit ratio metric
         :param random_seed: random seed
         :param loader_batch_size: batch size for dataloader
-        :param loader_shuffle: shuffle flag for dataloader
+        :param loader_shuffle: shuffle flag for train dataloader
```

Comment:
maybe replace by labelled instead of train to make it clear

Reply:
Changed